api/chaos/experimentation/v1/experimentation.proto

```diff
@@ -35,20 +36,25 @@ message CreateOrGetExperimentResponse {
   }
 
   // The created or fetched experiment that was requested by a caller. See `origin` to learn more
-  // about how to check whether a new experiment was created as the result of request's execution.
+  // about how to check whether a new experiment was created as the result of the request's execution.
   Experiment experiment = 1;
   // The type of returned experiment - either a newly created experiment or an existing one.
   Origin origin = 2;
 }
 
 message GetExperimentsRequest {
+  // The status of an experiment.
   enum Status {
+    // Unspecified status.
     STATUS_UNSPECIFIED = 0;
+    // Return experiments that are currently running.
     STATUS_RUNNING = 1;
   }
 
-  string config_type = 1;
-  Status status = 2;
+  // Return experiments whose config's type URL matches the provided config type URL.
+  string config_type = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  // Return experiments whose status is equal to the provided value.
+  Status status = 2 [ (validate.rules).enum.defined_only = true ];
```

Comment:
Defined only also allows setting UNSPECIFIED, do we want to allow that?

Reply:
added `not_int: 0` requirement